src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -2414,3 +2416,172 @@ def is_j_supersingular(j, proof=True):
     # expensive since it involves counting the number of points on E):
 
     return E.trace_of_frobenius() % p == 0
+
+def special_supersingular_curve(F, *, endomorphism=False):
+    r"""
+    Given a finite field ``F`` of degree `1` or `2`, construct a
+    "special" supersingular elliptic curve `E` defined over ``F``.
+
```

Comment:
It's not obvious to me while the degree has to be 1 or 2.

Reply:
To slightly simplify the implementation, mostly. I changed it to allow arbitrary finite fields.